ovos_config/models.py

```diff
@@ -90,9 +90,12 @@ def load_local(self, path=None):
                             config = yaml.safe_load(f)
                     else:
                         config = load_commented_json(path)
-                    for key in config:
-                        self.__setitem__(key, config[key])
-                    LOG.debug(f"Configuration {path} loaded")
+                    if config:
+                        for key in config:
+                            self.__setitem__(key, config[key])
+                        LOG.debug(f"Configuration {path} loaded")
+                    else:
+                        LOG.warning(f"Empty config found at: {path}")
```

Comment:
this should not be a warning, it is not an error or bad thing, can it be debug ?

Reply:
I wasn't sure if there's ever a case where an empty config file makes sense, but I can reduce to a debug since the load message is also a debug